nodes/src/nodes/llm_gmi/gmi_client.py

```
@@ -0,0 +1,103 @@
+        elif isinstance(error, RateLimitError):
+            return True
+        elif isinstance(error, APIConnectionError):
+            return True
+        else:
+            return False
+
+    def map_exception(self, error):
+        """Convert unfriendly exceptions to friendlier ones."""
+        if isinstance(error, AuthenticationError):
+            return ValueError('Invalid GMI Cloud API key.')
+        elif isinstance(error, APIError):
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)